app/code/community/MindArc/FatZebra/Model/Payment.php

```diff
@@ -112,11 +112,21 @@ public function capture(Varien_Object $payment, $amount)
                     $payment->setLastTransId($result->response->id);
                     $payment->setTransactionId($result->response->id);
                     $invoice = $order->getInvoiceCollection()->getFirstItem();
-                    if ($invoice && !$invoice->getEmailSent()) {
+                    if ($invoice) {
                         $invoice->pay(); // Mark the invoice as paid
                         $invoice->addComment("Payment made by Credit Card. Reference " . $result->response->id . ", Masked number: " . $result->response->card_number, false, true);
-                        $invoice->sendEmail();
-                        $invoice->save();
+                        /**
```

Comment:
As this is going into master and will be rolled out to clients without multi-shipping there should probably be a flag to only not sendEmail and save if multi-shipping is enabled.